airbyte-webapp/src/components/EntityTable/components/StatusCell.tsx

```diff
@@ -15,54 +13,21 @@ interface IProps {
   isManual?: boolean;
   id: string;
   onSync: (id: string) => void;
+  schemaChange: SchemaChange;
 }
 
-const ProgressMessage = styled.div`
-  padding: 7px 0;
-`;
-
-const StatusCell: React.FC<IProps> = ({ enabled, isManual, id, isSyncing, onSync, allowSync }) => {
-  const { mutateAsync: enableConnection, isLoading } = useEnableConnection();
-
-  const [{ loading }, OnLaunch] = useAsyncFn(async (event: React.SyntheticEvent) => {
-    event.stopPropagation();
-    onSync(id);
-  }, []);
-
-  if (!isManual) {
-    const onSwitchChange = async (event: React.SyntheticEvent) => {
-      event.stopPropagation();
-      await enableConnection({
-        connectionId: id,
-        enable: !enabled,
-      });
-    };
-
-    return (
-      // this is so we can stop event propagation so the row doesn't receive the click and redirect
-      // eslint-disable-next-line jsx-a11y/no-static-element-interactions
-      <div
-        onClick={(event: React.SyntheticEvent) => event.stopPropagation()}
-        onKeyPress={(event: React.SyntheticEvent) => event.stopPropagation()}
-      >
-        <Switch checked={enabled} onChange={onSwitchChange} disabled={!allowSync} loading={isLoading} />
-      </div>
-    );
-  }
-
-  if (isSyncing) {
-    return (
-      <ProgressMessage>
-        <FormattedMessage id="tables.progress" />
-      </ProgressMessage>
-    );
-  }
-
+export const StatusCell: React.FC<IProps> = ({ enabled, isManual, id, isSyncing, onSync, allowSync, schemaChange }) => {
   return (
-    <Button size="xs" onClick={OnLaunch} isLoading={loading} disabled={!allowSync || !enabled}>
-      <FormattedMessage id="tables.launch" />
-    </Button>
+    <div className={styles.container}>
+      <StatusCellControl
+        enabled={enabled}
+        id={id}
+        isSyncing={isSyncing}
+        isManual={isManual}
+        onSync={onSync}
+        allowSync={allowSync}
+      />
+      <ChangesStatusIcon schemaChange={schemaChange} />
```

Comment:
Need to check if the feature is enabled using the current flagging, for example:

```tsx
const isSchemaChangesFeatureEnabled = process.env.REACT_APP_AUTO_DETECT_SCHEMA_CHANGES = "true";

...
{isSchemaChangesFeatureEnabled && <ChangeStatusIcon ... />}
```